### PR TITLE
samples: ucaip samples batch 6 of  6

### PR DESCRIPTION
**Pull request by munkhuushmgl**

### CODE REVIEW

File: samples/snippets/src/main/java/aiplatform/CreateTrainingPipelineTextClassificationSample.java [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+      System.out.format(
+          "\t\t\t\tFeature Attributions Schema_uri: %s\n",
+          explanationMetadata.getFeatureAttributionsSchemaUri());
+
+      Status status = trainingPipelineResponse.getError();
+      System.out.println("\tError");
+      System.out.format("\t\tCode: %s\n", status.getCode());
+      System.out.format("\t\tMessage: %s\n", status.getMessage());
+
+      List<Any> detailsList = status.getDetailsList();
```
Comment (stephaniewang526) [inline samples/snippets/src/main/java/aiplatform/CreateTrainingPipelineTextClassificationSample.java:233]: Should we print out `detailsList`? or its attributes?

File: samples/snippets/src/main/java/aiplatform/CreateTrainingPipelineTextEntityExtractionSample.java [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+      System.out.format(
+          "\t\t\t\tFeature Attributions Schema_uri: %s\n",
+          explanationMetadata.getFeatureAttributionsSchemaUri());
+
+      Status status = trainingPipelineResponse.getError();
+      System.out.println("\tError");
+      System.out.format("\t\tCode: %s\n", status.getCode());
+      System.out.format("\t\tMessage: %s\n", status.getMessage());
+
+      List<Any> detailsList = status.getDetailsList();
```
Comment (stephaniewang526) [inline samples/snippets/src/main/java/aiplatform/CreateTrainingPipelineTextEntityExtractionSample.java:234]: Same as above - should we print this out?

File: samples/snippets/src/main/java/aiplatform/CreateTrainingPipelineTextSentimentAnalysisSample.java [hunk 3]
```diff
@@ -0,0 +1,239 @@
...
+    // once, and can be reused for multiple requests. After completing all of your requests, call
+    // the "close" method on the client to safely clean up any remaining background resources.
+    try (PipelineServiceClient pipelineServiceClient =
+        PipelineServiceClient.create(pipelineServiceSettings)) {
+      String location = "us-central1";
+      String trainingTaskDefinition =
+          "gs://google-cloud-aiplatform/schema/trainingjob/definition/"
+                  + "automl_text_sentiment_1.0.0.yaml";
+
+      // Use sentimentMax of 4
```
Comment (stephaniewang526) [inline samples/snippets/src/main/java/aiplatform/CreateTrainingPipelineTextSentimentAnalysisSample.java:79]: Is this a recommendation or just a description? If it is a recommendation, can we add a bit more context on why we should use sentimentMax of 4? If it's a description then can we link to sentimentMax doc?

File: samples/snippets/src/main/java/aiplatform/CreateTrainingPipelineTextSentimentAnalysisSample.java [hunk 4]
```diff
@@ -0,0 +1,239 @@
...
+      System.out.format(
+          "\t\t\t\tFeature Attributions Schema_uri: %s\n",
+          explanationMetadata.getFeatureAttributionsSchemaUri());
+
+      Status status = trainingPipelineResponse.getError();
+      System.out.println("\tError");
+      System.out.format("\t\tCode: %s\n", status.getCode());
+      System.out.format("\t\tMessage: %s\n", status.getMessage());
+
+      List<Any> detailsList = status.getDetailsList();
```
Comment (stephaniewang526) [inline samples/snippets/src/main/java/aiplatform/CreateTrainingPipelineTextSentimentAnalysisSample.java:235]: Same as above - should we print this out?

File: samples/snippets/src/main/java/aiplatform/ImportDataTextClassificationSingleLabelSample.java [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+                  .build());
+
+      OperationFuture<ImportDataResponse, ImportDataOperationMetadata> importDataResponseFuture =
+          datasetServiceClient.importDataAsync(datasetName, importDataConfigList);
+      System.out.format(
+          "Operation name: %s\n", importDataResponseFuture.getInitialFuture().get().getName());
+
+      System.out.println("Waiting for operation to finish...");
+      ImportDataResponse importDataResponse = importDataResponseFuture.get(300, TimeUnit.SECONDS);
+      System.out.format("Import Data Text Classification Response: %s\n", importDataResponse);
```
Comment (stephaniewang526) [inline samples/snippets/src/main/java/aiplatform/ImportDataTextClassificationSingleLabelSample.java:85]: Do we need a `toString()` here for `importDataResponse`?

File: samples/snippets/src/main/java/aiplatform/ImportDataTextEntityExtractionSample.java [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+                  .build());
+
+      OperationFuture<ImportDataResponse, ImportDataOperationMetadata> importDataResponseFuture =
+          datasetServiceClient.importDataAsync(datasetName, importDataConfigList);
+      System.out.format(
+          "Operation name: %s\n", importDataResponseFuture.getInitialFuture().get().getName());
+
+      System.out.println("Waiting for operation to finish...");
+      ImportDataResponse importDataResponse = importDataResponseFuture.get(300, TimeUnit.SECONDS);
+      System.out.format("Import Data Text Entity Extraction Response: %s\n", importDataResponse);
```
Comment (stephaniewang526) [inline samples/snippets/src/main/java/aiplatform/ImportDataTextEntityExtractionSample.java:84]: Same as above -- consider `toString()`?

**Review by stephaniewang526 — CHANGES_REQUESTED** [review]

File: samples/snippets/src/main/java/aiplatform/CreateTrainingPipelineTextSentimentAnalysisSample.java (hunk 3 above)
Comment (munkhuushmgl) [reply samples/snippets/src/main/java/aiplatform/CreateTrainingPipelineTextSentimentAnalysisSample.java]: I just added detailed explanation on sentiment max

**Review by stephaniewang526 — APPROVED** [review]

**Pull request merged**